deployments/helm/sumologic-the-coffee-bar/values.yaml

```diff
@@ -248,7 +304,34 @@ nodeSelector: {}
 
 tolerations: []
 
-affinity: {}
+affinity: {} 
+
+postgres_affinity: 
```

Comment:
Can't accept affinities based on custom namespaces. Please leave it as generic `postgres_affinity: {}` and apply needed changes in your `custom-values.yaml`